versioned_docs/version-v0.53/tutorials/proposals/freeform-proposal.md

```diff
@@ -21,8 +21,8 @@ import TabItem from '@theme/TabItem';
 # Create a freeform proposal
 The aim of this is to allow community to provide votes on proposals which don't change any of the behaviour of the currently running Vega blockchain. That is to say, at enactment time, no changes are effected on the system, but the record of how token holders voted will be stored on chain. 
 
-Note that to create a freeform proposal you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minProposerBalance" hideName={true} suffix="tokens" />
-, while to vote you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minVoterBalance" suffix="tokens" hideName={true}/>.
+Note that to create a freeform proposal you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minProposerBalance"  formatter='governanceToken' hideName={true} suffix="tokens" />
+, while to vote you need <NetworkParameter frontMatter={frontMatter} param="governance.proposal.freeform.minVoterBalance" suffix="tokens" formatter='governanceToken' hideName={true}/>.
```

Comment:
It looks like the formatter for gov token isn't working as it shows as 0, rather than "anything more than 0" in this case

Reply:
Fixed. The code originall was like IF this is a token value AND the token value is 0.00000000...1, THEN display 'more than 0'. This network parameter is *actually* 0. So now there's another special case that ends up displaying '0 or more' versus 'more than 0' versus '0.000002' or whatever